web/src/interfaces.ts

```diff
@@ -2,3 +2,11 @@ export interface Image {
   url: string;
   alt?: string;
 }
+
+export interface Customer {
+  id: number;
+  gpayId: string; // E164 format
```

Comment:
I guess you want this for the contact number

Reply:
@chshda Actually, client side don't need this value, this is the value that client side have to provide server side when creating or logging in a customer. Maybe I will just remove this field from the get response in another PR!